string name
string color
---
int32 completed_flag